roles/util-cfg-service/templates/ui/de.properties.j2

```diff
@@ -204,3 +204,11 @@ org.iplantc.discoveryenvironment.notification.amqp.exchange.auto-delete = {{ amq
 ###############################################################################
 org.iplantc.discoveryenvironment.analysis.support.user = siuser
 org.iplantc.discoveryenvironment.support = {{ terrain.base }}/support-email
+
+###############################################################################
+# DE Session Store Settings
+###############################################################################
+org.iplantc.discoveryenvironment.redis.host = {{ redis_host }}
+org.iplantc.discoveryenvironment.redis.port = {{ redis_port }}
+org.iplantc.discoveryenvironment.redis.db.number = 0
+org.iplantc.discoveryenvironement.redis.password = {{ redis_password }}
```

Comment:
Good catch on the typo. Will get that fixed. Will also add them to `de.properties.tmpl`